### PR TITLE
Handle View Exceptions

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: apps/quickbooks_online/views.py [hunk 1]
```diff
@@ -47,28 +38,21 @@ def get_queryset(self):
         return DestinationAttribute.objects.filter(
             attribute_type='VENDOR', active=True, workspace_id=self.kwargs['workspace_id']).order_by('value')[:10]
 
+    @handle_view_exceptions(task_name="Post Get Vendors from QBO")
     def post(self, request, *args, **kwargs):
```
Comment (ashwin1111) [inline apps/quickbooks_online/views.py:42]: post method for this endpoint might not be used, please check and remove

File: apps/mappings/views.py [hunk 2]
```diff
@@ -17,45 +18,37 @@ class AutoMapEmployeeView(generics.CreateAPIView):
...
+    @handle_view_exceptions(task_name="Post Trigger Auto Map employees")
     def post(self, request, *args, **kwargs):
         """
         Trigger Auto Map employees
         """
-        try:
-            workspace_id = kwargs['workspace_id']
-            general_settings = WorkspaceGeneralSettings.objects.get(workspace_id=workspace_id)
+        workspace_id = kwargs['workspace_id']
+        general_settings = WorkspaceGeneralSettings.objects.get(workspace_id=workspace_id)
```
[the remark this answers is not in the document]
Comment (ruuushhh) [reply apps/mappings/views.py:27]: Can you explain more?

File: apps/quickbooks_online/views.py (hunk 1 above)
Comment (ruuushhh) [reply apps/quickbooks_online/views.py:42]: Yes we are not using this post method

File: apps/quickbooks_online/views.py [hunk 3]
```diff
@@ -86,58 +70,42 @@ def get_queryset(self):
         return DestinationAttribute.objects.filter(
             attribute_type='EMPLOYEE', active=True, workspace_id=self.kwargs['workspace_id']).order_by('value')[:10]
 
+    @handle_view_exceptions(task_name="Post Get Employees from QBO")
```
[the remark this answers is not in the document]
Comment (ruuushhh) [reply apps/quickbooks_online/views.py:73]: we are not using this as well
Removing both from code

File: apps/quickbooks_online/views.py [hunk 4]
```diff
@@ -70,7 +54,7 @@ def get_queryset(self):
         return DestinationAttribute.objects.filter(
             attribute_type='EMPLOYEE', active=True, workspace_id=self.kwargs['workspace_id']).order_by('value')[:10]
 
-    @handle_view_exceptions(task_name="Post Get Employees from QBO")
+    @handle_view_exceptions()
```
Comment (ashwin1111) [inline apps/quickbooks_online/views.py:57]: post method might not be used, check and remove

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**